tests/e2e/e2e_test.go

```diff
@@ -0,0 +1,123 @@
+package e2e
+
+import (
+	"bytes"
+	"context"
+	"log"
+	"os/exec"
+
+	"github.com/firepear/qsplit"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/xridge/kubestone/api/v1alpha1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	k8sscheme "k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+	ctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	perfv1alpha1 "github.com/xridge/kubestone/api/v1alpha1"
+)
+
+const (
+	iperf3SampleCR = "../../config/samples/perf_v1alpha1_iperf3.yaml"
+	e2eNamespace   = "kubestone-e2e"
+)
+
+var restClientConfig = ctrl.GetConfigOrDie()
+var client ctrlclient.Client
+var ctx = context.Background()
+var scheme = runtime.NewScheme()
+
+func init() {
+	_ = k8sscheme.AddToScheme(scheme)
+	err := perfv1alpha1.AddToScheme(scheme)
+	if err != nil {
+		panic(err)
+	}
+
+	client, err = ctrlclient.New(restClientConfig, ctrlclient.Options{Scheme: scheme})
+	if err != nil {
+		panic(err)
+	}
+}
+
+func run(command string) (stdout, stderr string, err error) {
```

Comment:
It is not a testcase...
Would it make sense to move this into an other file?

Reply:
done, moved it to `utils.go`